.gitlab-ci-check-docker-acceptance.yml

```
@@ -133,9 +133,10 @@ publish:acceptance:
     #  According to https://docs.coveralls.io/supported-ci-services
     #  we should set CI_NAME, CI_BUILD_NUMBER, etc. But according
     #  to goveralls source code (https://github.com/mattn/goveralls)
-    #  many of these are not supported. Set only CI_PR_NUMBER and
-    #  pass few others are command line arguments.
+    #  many of these are not supported. Set CI_BRANCH, CI_PR_NUMBER,
+    #  and pass few others are command line arguments.
```

```suggestion
    #  and pass few others as command line arguments.
```
Since we're already on it